src/Containers/TableFeature/TableFeature.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { createStoryTitle } from '../../Constants';
+import { TableFeature, TableFeatureProps } from './TableFeature'; 
+
+const data = [ 
+    {
+        colOne: 'Uploading payout data',  
+        colTwo: 'Manual', 
+        colThree: 'Automatic',
+    },
+    {
+        colOne: 'Verifying recipient data',  
+        colTwo: 'Manual', 
+        colThree: 'Automatic',
+    },
+    {
+        colOne: 'Issuing payments',  
+        colTwo: 'Manual', 
+        colThree: 'Automatic',
+    },
+    {
+        colOne: 'Reporting payouts',  
+        colTwo: 'Manual', 
+        colThree: 'Automatic',
+    },
+    {
+        colOne: 'Tracking balances',  
+        colTwo: 'Manual', 
+        colThree: 'Automatic',
+    },
+    {
+        colOne: 'Error handling',  
+        colTwo: 'Manual', 
+        colThree: 'Automatic',
+    },
+];
+
+export default {
+    title: createStoryTitle('TableFeature'),
+    component: TableFeature,
+    args: {
+        headingOne: 'Simplified payout process',
+        headingTwo: 'Automate payout workflows',
+        title:      'Automated payouts',
+        paragh:     'Stop uploading spreadsheets and juggling multiple payout providers. Use a single integration to automate hours of operational overhead and reduce opportunities for human error.',
```

Comment:
becuase this is a React UI Library, The component you create will need to be extremely Clear and Readable. <TableFeature by the props what goes where?